Makefile.am

```diff
@@ -121,6 +126,72 @@ CLEANFILES += $(eos_updater_avahi_in:%.in=%)
 
 dist_man8_MANS += eos-updater-avahi/docs/eos-updater-avahi.8
 
+# Server utility library
+noinst_LTLIBRARIES += libeos-update-server/libeos-update-server-@EUS_API_VERSION@.la
+
+libeos_update_server_libeos_update_server_@EUS_API_VERSION@_la_CPPFLAGS = \
+	$(AM_CPPFLAGS) \
+	$(NULL)
+libeos_update_server_libeos_update_server_@EUS_API_VERSION@_la_CFLAGS = \
+	$(CODE_COVERAGE_CFLAGS) \
+	$(WARN_CFLAGS) \
+	$(EOS_UPDATE_SERVER_CFLAGS) \
+	$(NULL)
+libeos_update_server_libeos_update_server_@EUS_API_VERSION@_la_LDFLAGS = \
+	$(WARN_LDFLAGS) \
+	$(NULL)
+libeos_update_server_libeos_update_server_@EUS_API_VERSION@_la_LIBADD = \
+	$(CODE_COVERAGE_LIBS) \
+	$(top_builddir)/libeos-updater-util/libeos-updater-util-@EUU_API_VERSION@.la \
+	$(EOS_UPDATE_SERVER_LIBS) \
+	$(NULL)
+libeos_update_server_libeos_update_server_@EUS_API_VERSION@_la_SOURCES = \
+	libeos-update-server/config.c \
+	libeos-update-server/config.h \
+	libeos-update-server/repo.c \
+	libeos-update-server/repo.h \
+	libeos-update-server/server.c \
+	libeos-update-server/server.h \
+	$(NULL)
+
+# eos-update-server program
+libexec_PROGRAMS += eos-update-server/eos-update-server
```

Comment:
Yeah, the historic installation locations of all the binaries are all over the place. This branch doesn’t change the installation location for `eos-update-server`. I don’t place a high priority on changing those locations at the moment, as it would undoubtedly cause breakage.